cpp/src/strings/findall.cu

```diff
@@ -164,29 +164,28 @@ std::unique_ptr<table> findall_re(
       strings_count));
 
   for (int32_t column_index = 0; column_index < columns; ++column_index) {
-    rmm::device_vector<string_index_pair> indices(strings_count);
-    string_index_pair* d_indices = indices.data().get();
+    rmm::device_uvector<string_index_pair> indices(strings_count, stream);
```

Comment:
I'm making some more involved changes in this file in #7428

Reply:
Resolved.